lib/taskana-core-test/src/test/java/acceptance/task/query/TaskQueryImplAccTest.java

```diff
@@ -262,6 +260,7 @@ void should_FilterByExternalId_When_QueryingForExternalIdNotIn() {
     @Nested
     @TestInstance(Lifecycle.PER_CLASS)
     class Received {
+
```

Comment:
Did you add this manually, or did you IDE format this on its own?

Reply:
The IDE formatted it like this. Should I format it back manually?

Reply:
Ok :)